docs/features/teams.rst

```diff
@@ -0,0 +1,85 @@
+.. _teams:
+
+Teams
+=====
+
+Pootle added the team page which is meant to ease management of team members
+and their roles within the team, and other team related aspects and features.
+The team page also provides overall statistics and access to the bulk handling
+for pending suggestions.
+
+.. image:: ../_static/accessing_team.png
+
+Language administrators will find the team interface on the first dropdown in
+their language's browse page.
+
+The team page is divided in two parts, on the left the management of the team
+members and on the right are listed the language's projects, and the overall
+translation statistics with links to the editor and the bulk suggestions
+manager.
+
+
+.. _teams#manage-members:
+
+Managing team members
+---------------------
+
+The UI for team members management is divided in two parts: on the upper side
+there is a form to add a new member, and below the team members are listed in
```

Comment:
Use numbered bullets.

1. Top left - a form to add new members
2. Bottom left - the teams members listed within their assigned roles